pkgs/by-name/kn/knxd/package.nix

```diff
@@ -35,8 +36,12 @@ stdenv.mkDerivation rec {
     ++ lib.optional withUsb libusb1
     ++ lib.optional stdenv.isDarwin argp-standalone;
 
-  configureFlags = lib.optional (!withSystemd) "--disable-systemd"
-    ++ lib.optional (!withUsb) "--disable-usb";
+  configureFlags = [
+    (lib.enableFeature withSystemd "systemd")
+    (lib.enableFeature withUsb "usb")
+  ];
+
+  CC_FOR_BUILD="${buildPackages.stdenv.cc}/bin/cc";
```

Comment:
The proper way is to `depsBuildBuild = [ buildPackages.stdenv.cc ];` to populate the vars

Reply:
Good point, thanks!